scripts/deploy.ts

```diff
@@ -374,6 +376,20 @@ function buildCommonPost(buildObj: BuildObj) {
     )
     mylog('\n*** Upload to App Center Complete ***')
   }
+
+  if (buildObj.rsyncLocation != null) {
+    mylog(`\n\nUploading to rsyncLocation ${buildObj.rsyncLocation}`)
+    mylog('***********************\n')
+
+    // const rsyncDir = join(buildObj.rsyncLocation, repoBranch, platformType, String(buildNum))
+
+    const datePrefix = new Date().toISOString().slice(2, 19).replace(/:/gi, '').replace(/-/gi, '')
+    const rsyncFile = escapePath(`${datePrefix}--${productNameClean}--${platformType}--${repoBranch}--${buildNum}--${guiHash.slice(0, 8)}.ipa`)
+
+    // call(`rsync -avz -e "ssh -i ${githubSshKey}" ${buildObj.ipaFile} ${rsyncDir}/`)
+    call(`rsync -avz -e "ssh -i ${githubSshKey}" ${buildObj.ipaFile} ${(join(buildObj.rsyncLocation), rsyncFile)}`)
+    mylog('\n*** Upload to rsyncLocation Complete ***')
+  }
```

Comment:
You should co-locate line 328 within his section of code (`const { buildNum, guiHash, platformType, productNameClean, repoBranch } = buildObj`). This destructuring is for line 387, so maybe put it above that line?